From d52cbb52f27d046aaef4c35cd52d0c95544ecb97 Mon Sep 17 00:00:00 2001
From: Le Ma <le.ma@amd.com>
Date: Sun, 30 Jun 2019 11:35:32 +0800
Subject: [PATCH 2978/4256] drm/amdgpu: limit sdma instances to 2 for Arcturus
 in BU phase

Another 6 sdma instances do not work at present. Disable them to unblock KFD
for silicon bringup as a workaround

Signed-off-by: Le Ma <le.ma@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c b/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
index 48d4597ef9f6..9c7a3eb9a9a0 100644
--- a/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
+++ b/drivers/gpu/drm/amd/amdgpu/sdma_v4_0.c
@@ -1601,7 +1601,7 @@ static int sdma_v4_0_early_init(void *handle)
 	if (adev->asic_type == CHIP_RAVEN)
 		adev->sdma.num_instances = 1;
 	else if (adev->asic_type == CHIP_ARCTURUS)
-		adev->sdma.num_instances = 8;
+		adev->sdma.num_instances = 2;
 	else
 		adev->sdma.num_instances = 2;
 
-- 
2.17.1

